client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -494,7 +494,9 @@ export const siteSetupFlow: Flow = {
 					return exitFlow( `/home/${ siteSlug }` );
 
 				case 'import':
-					return navigate( 'importList' );
+					return isEnabled( 'onboarding/import-light' )
+						? navigate( 'intent' )
```

Comment:
Depends on the timing as the Non-English users will move over to Goals Step soon now that there's a CFT in progress. But anyway I see this line is reverted.

Reply:
Yeah, this line is reverted because the skip link will not appear on the new URL capture screen and we don't need to define its `goNext` action here anymore.